tests/lib/rules/consistent-return.js

```diff
@@ -207,7 +256,10 @@ ruleTester.run("consistent-return", rule, {
                     messageId: "missingReturn",
                     data: { name: "function" },
                     type: "FunctionExpression",
-                    column: 3
+                    line: 1,
+                    column: 3,
+                    endLine: 1,
+                    endColumn: 37
```

Comment:
Reporting the entire function including its body seems inconsistent here. Everywhere else we have missing returns, the rule selects the function name, `function` keyword, or `=>` token. I'd expect this case to stop at the end of the `function` keyword like the rest.

Reply:
Hmm, should stop at tge end of `function`, I will update 👍